app/v1/models/response/liability/TaxPeriod.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response.liability
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+import utils.NestedJsonReads
+
+case class TaxPeriod(from: String, to: String)
+
+object TaxPeriod extends NestedJsonReads {
```

Comment:
I don't think this needs to extend NestedJsonReads. It doesn't appear to be using it at least.